tests/unit/ScoreCalc.spec.js

```diff
@@ -24,90 +22,115 @@ describe('ScoreCalc.vue', () => {
     break: 4,
   };
   vm.playerPerformance = {
-    tap: { critical: 183, better_perfect: 0, perfect: 219, great: 29, good: 1, miss: 0 },
-    hold: { critical: 16, better_perfect: 0, perfect: 16, great: 0, good: 0, miss: 0 },
-    slide: { critical: 158, better_perfect: 0, perfect: 0, great: 6, good: 2, miss: 0 },
-    touch: { critical: 0, better_perfect: 0, perfect: 0, great: 0, good: 0, miss: 0 },
-    break: { critical: 2, better_perfect: 2, perfect: 0, great: 0, good: 0, miss: 0 },
+    tap: {
+      critical: 183,
+      perfect: 219,
+      great: 29,
+      good: 1,
+      miss: 0,
+    },
+    hold: {
+      critical: 16,
+      perfect: 16,
+      great: 0,
+      good: 0,
+      miss: 0,
+    },
+    slide: {
+      critical: 158,
+      perfect: 0,
+      great: 6,
+      good: 2,
+      miss: 0,
+    },
+    touch: {},
```

Comment:
I skipped some zero values. The rest of this test file is all changed by formatter 😓

Reply:
Try setting "editor.formatOnSave" to false in vscode settings.